__tests__/app.test.js

```diff
@@ -218,5 +218,37 @@ describe('nc-be-news-app', () => {
       // Check the amount of items in the object is correct
       expect(Object.keys(comment).length).toBe(6);
```

Comment:
You can also check to see what you are getting back in the comment body.

Reply:
done